data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -163,9 +186,20 @@
     field_names.FPL_200_AND_COLLEGE_ATTENDANCE_SERIES: "FPL200S",
     field_names.THRESHOLD_COUNT: "TC",
     field_names.ISLAND_AREAS_UNEMPLOYMENT_LOW_HS_EDUCATION_FIELD: "IAULHSE",
-    field_names.ISLAND_AREAS_POVERTY_LOW_HS_EDUCATION_FIELD: "ISPLHSE",
+    field_names.ISLAND_AREAS_POVERTY_LOW_HS_EDUCATION_FIELD: "IAPLHSE",
```

Comment:
I think this tile short name actually had a typo

Reply:
If we're updating it, let's make sure to flag it for @vim-usds on the front-end dictionary.